crates/nu-cli/src/util.rs

```diff
@@ -237,14 +351,16 @@ pub fn eval_source(
     exit_code
 }
 
-fn evaluate_source(
+pub fn eval_source(
```

Comment:
Any reason for the name change? This was intended I think to have a similar name to e.g. `evaluate_file()`, `evaluate_repl()`

Reply:
We previously had `eval_source()` which returned `i32` and was called from other places, and `evaluate_source()` which returned `Result<(), ShellError>` and was only called internally from `eval_source()`. So I kept it with that name to avoid having to change all the other places where it's called.